www/data_module/src/specification.constant.coffee

```diff
@@ -1,272 +1,280 @@
-                'buildslaveid'
-                'configured_on'
-                'connected_to'
-                'name'
-                'slaveinfo'
-            ]
-            root: true
-            paths: []
-            static: true
-        changes:
-            id: 'changeid'
-            fields: [
-                'changeid'
-                'author'
-                'branch'
-                'category'
-                'codebase'
-                'comments'
-                'files'
-                'parent_changeids'
-                'project'
-                'properties'
-                'repository'
-                'revision'
-                'revlink'
-                'sourcestamp'
-                'when_timestamp'
-            ]
-            root: true
-            paths: []
-            static: true
-        changesources:
-            id: 'changesourceid'
-            fields: [
-                'changesourceid'
-                'master'
-                'name'
-            ]
-            root: true
-            paths: []
-            static: true
-        forceschedulers:
-            id: 'name'
-            fields: [
-                'name'
-                'all_fields'
-                'builder_names'
-                'label'
-            ]
-            root: true
-            paths: []
-            static: true
-        masters:
-            id: 'masterid'
-            fields: [
-                'masterid'
-                'active'
-                'last_active'
-                'name'
-            ]
-            root: true
-            paths: [
-                'builders'
-                'builders/n:builderid'
-                'builders/n:builderid/buildslaves'
-                'builders/n:builderid/buildslaves/n:buildslaveid'
-                'builders/n:builderid/buildslaves/i:name'
-                'buildslaves'
-                'buildslaves/i:name'
-                'buildslaves/n:buildslaveid'
-                'changesources'
-                'changesources/n:changesourceid'
-                'schedulers'
-                'schedulers/n:schedulerid'
-            ]
-            static: true
-        schedulers:
-            id: 'schedulerid'
-            fields: [
-                'schedulerid'
-                'master'
-                'name'
-            ]
-            root: true
-            paths: []
-            static: true
-        sourcestamps:
-            id: 'ssid'
-            fields: [
-                'ssid'
-                'branch'
-                'codebase'
-                'created_at'
-                'patch'
-                'project'
-                'repository'
-                'revision'
-            ]
-            root: true
-            paths: [
-                'changes'
-            ]
-            static: true
+            builds:
+                id: 'buildid'
+                fields: [
+                    'buildid'
+                    'builderid'
+                    'buildrequestid'
```

Comment:
Why is it important to describe all fields in this spec?

We should have a way to make sure the spec here is coherent with the dataspec generated by buildbot dataspec.
We should not reproduce the mistake of verifying it within the normal build, but at least make sure it is automated in the CI (.travisrc)